MSET9_installer_script/mset9.bat

```diff
@@ -1,12 +1,47 @@
-@echo off
-chcp 65001 > nul
-py -V > nul
-if %errorlevel% NEQ 0 (
-    echo Python 3 is not installed.
-    echo Please install Python 3 and try again.
-    echo https://www.python.org/downloads/
-    echo.
-    pause
-    exit
-)
-py -3 mset9.py
\ No newline at end of file
+@echo off
+setlocal enabledelayedexpansion
+chcp 65001 > nul
+
+py -V > nul 2>&1
+if %errorlevel% EQU 0 py -3 mset9.py & exit /B
+python -V > nul 2>&1
+if %errorlevel% EQU 0 python mset9.py & exit /B else (

```

Comment:
This syntax is disgusting.. Could you reformat it like so:

```
if %errorlevel% EQU 0 (
  <stuff here>
)
else (
  <stuff here>
)
```

Having an else on the same line as an if provides poor readability. I see this come up a few times in your changes. I know you're saving a couple lines with this syntax but it makes it hard to maintain for future contributors

Reply:
Fair enough.